activiti-cloud-runtime-bundle-service/activiti-cloud-services-runtime-bundle/activiti-cloud-services-core/src/main/java/org/activiti/cloud/services/core/BaseProcessDefinitionService.java

```diff
@@ -69,13 +70,25 @@ protected ExtendedCloudProcessDefinition decorate(
     }
 
     protected GetProcessDefinitionsPayload buildGetProcessDefinitionsPayload(String excludedCategory) {
+        var processDefinitionsPayloadBuilder = getGetProcessDefinitionsPayloadBuilder(excludedCategory);
+        return processDefinitionsPayloadBuilder.build();
+    }
+
+    private GetProcessDefinitionsPayloadBuilder getGetProcessDefinitionsPayloadBuilder(String excludedCategory) {
         var processDefinitionsPayloadBuilder = ProcessPayloadBuilder.processDefinitions();
         if (validateInput(excludedCategory)) {
             LOGGER.debug("Excluding process definitions with category: {}", excludedCategory);
 
             processDefinitionsPayloadBuilder.withProcessCategoryToExclude(excludedCategory);
         }
-        return processDefinitionsPayloadBuilder.build();
+        return processDefinitionsPayloadBuilder;
+    }
+
+    protected GetProcessDefinitionsPayload buildGetProcessDefinitionsPayloadWithLatestVersion(
+        String excludedCategory,
+        boolean latestVersions
```

Comment:
It should be `latestVersion` (singular) as only the latest version will be returned for each process definition.